### DISCUSSION TITLE
Error in vtk conversion from animation file

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1477] [Q&A] [answered]
Hi, I'm trying to convert the animation file (.h3d) to the vtk estentions to be able to view the results on paraview but it cannot open the file or either give me this error "Error in Anim Files version ".
I'm running this script:
cd C:\02_open_radioss\test
C:\02_open_radioss\OpenRadioss\exec\anim_to_vtk_win64.exe BIRD_WINDSHIELD_v1.h3d > BIRD_WINDSHIELD_v1_001.vtk
The files generated in the directory of the input file are the following:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/a498a94e-9c66-4cdb-8ab3-0add56c9e6fd)

**Comment by sebastienVilleneuve** [comment]
Hi @FGarelli 

Animation files than can be used with converter are A00x files only.
.h3d file is not supported.

Thanks,
Sebastien

**Reply by FGarelli** [reply]
Hi Sebastien, these are the only files that were generated by Openradioss.
I'm using the Windows version. 
Do you know how i can get the A00x files? Or do you know any ways to convert the h3d files to be view in Paraview?

**Comment by sebastienVilleneuve** [comment, marked as answer]
Hi @FGarelli 

in 1.rad file , you need to output the Anim file(s) with /ANIM/DT option and then select the results you want to have in the file :
https://help.altair.com/hwsolvers/rad/topics/solvers/rad/animation_post_process_output_files_engine_r.htm

Regards,
Sébastien
